.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,16 +34,20 @@ A detailed explanation of this checklist is available here:
 https://github.com/nvaccess/nvda/blob/master/devDocs/githubPullRequestTemplateExplanationAndExamples.md#code-review-checklist
 -->
 
-- [ ] Pull Request description is up to date.
-- [ ] Unit tests.
-- [ ] System (end to end) tests.
-- [ ] Manual testing.
-- [ ] API is compatible with existing addons.
-- [ ] User Documentation.
-- [ ] Change log entry.
-- [ ] Context sensitive help for GUI changes.
+- [ ] Pull Request description:
+  - description is up to date
+  - change log entries
+- [ ] Testing:
+  - Unit tests
+  - System (end to end) tests
+  - Manual testing
+- [ ] API is compatible with existing add-ons.
+- [ ] Documentation:
+  - User Documentation
```

Comment:
"Developer documentation" should also be updated if necessary.

```suggestion
  - User Documentation
  - Developer / Technical Documentation
```

Reply:
Checklist: Dev doc item addition addressed in 2dbdc6e4b4e9c164cdfd46acf38cdde4cc4f9bde and in 834d98d5d518541eeeb65fa1cbc7f1dc7be67715.